docs/extensions-plugins/neovim/quickstart.mdx

```diff
@@ -29,7 +29,7 @@ A few things to note before you start installing the Pieces Neovim Plugin:
 - The Python based Neovim plugins requires the `pynvim` Python package, which is not included in the default Neovim installation. You can install it by running the following command:
 
     ```bash
-    pip install pynvim
+    sudo apt install python3-pynvim
```

Comment:
Hey @Debajyati thanks for your contribution, I will recommend that we can add this in an info section in the docs since not everyone will use a Debian based system. For mac users they can still run the current command. 
```
::: info
:::
```

Reply:
Okay I didn't know about mac because I never used one. But I know that it is not just Debian based systems. In any linux distribution you would need to manually install any python package with the system package manager( unless you are inside a virtual environment and adding the libraries as dependency). Like in Arch Linux the command would be - 

```shell
sudo pacman -S python3-pynvim
```

the system package manager would change with the flags/options but the package would always be `python3-pynvim`.   
I mentioned Debian Based because Ubuntu is debian based and I read somewhere in the Docs that Pieces currently supports only Ubuntu and all of its derivatives. So all of the supported systems share apt as the package manager.   
And the largest user base of Neovim are Linux Users -   
 

![](https://github.com/user-attachments/assets/8d8c8bc9-8b8b-414d-a93f-8bebc191376a)

So, hey! @shivay-at-pieces I think that it would be better if we make it a two part section.   
For MacOS users - 

```
pip install pynvim
```

If you use Debian/Ubuntu - 

```
sudo apt install python3-pynvim
```

What do you think?

Reply:
Yeah this makes sense @Debajyati